tests/qm-sanity-test/test.sh

```diff
@@ -55,26 +82,28 @@ Exec=sleep 1000
 WantedBy=multi-user.target default.target
 EOF
 
-    info_message "check_podman_run_and_exec(): qm-sanity-test reload & restart"
+    info_message "check_qm_podman_quadlet_is_ok(): qm-sanity-test reload & restart"
     exec_cmd "podman exec qm systemctl daemon-reload"
     exec_cmd "podman exec qm systemctl start qm-sanity-test"
     exec_cmd "podman exec qm systemctl status qm-sanity-test | grep -i started"
 
-    exec_cmd "podman exec qm podman run alpine echo hello"
+    exec_cmd "podman exec qm podman run alpine echo Hello QM"
 }
 
 # Verify /var partition exist
 check_var_partition_exist(){
-    if [[ "$(podman exec -it qm bash -c "ls /var")" =~ "No such file or directory" ]]; then
-        info_message "check_var_partition_exist: /var does not exist inside QM"
-        exit 1
+    if [[ "$(lsblk)" =~ "part /usr/lib/qm/rootfs/var" ]]; then
```

Comment:
some thing like this 
[[ "$(lsblk -o 'MAJ:MIN,TYPE,MOUNTPOINTS')" =~ "part /var" ]]

Reply:
done